sculture-rest/src/main/java/sculture/dao/UserDao.java

```diff
@@ -20,10 +20,7 @@
      * Save the user in the database.
      */
     public void create(User user) {
-        if (getByEmail(user.getEmail()) != null) {
-            entityManager.persist(user);
-        } else throw new UserAlreadyExistsException();
-        return;
+        entityManager.persist(user);
```

Comment:
(getByEmail(user.getEmail()) throws error, does not returns null.
The uniqueness of email adreses is handling by database.
Previously the email addresses was marked as unique on table model but they was not unique on database. The reason is that the unique constraint on model is added later. Thus the hibernate did not update the database.
I have manually added the unique constraint on database. For new databases the hibernate will add unique constraint.